org-formation/600-access/bedrock-agent-role.yaml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This is the role used by the agent when it's running. It has the right to execute a model.